src/utils/contribution.js

```diff
@@ -81,12 +81,16 @@ export default class Contribution {
   }
 
   /**
+   *
    * Applies changes to the specific component
+   * @static
    * @param  {} component original component
    * @param  {} changes object containing changes
    * @param  {} field field to check
    * @param  {} value value to apply to the field
    * @param  {} type used to specify a type of field (e.g. array) to perform different kind of operations
+   * @param {*} [transform=a => a]
```

Comment:
What is this new @param {*} for? 
Make sure to add also a comment specifying its usage (in the code)

Reply:
I'm not the correct convention. But I've replaced `{*}`with `{}` for consistency.

I've also added a comment as you correctly pointed out